recipes/obspy/run_test.py

```diff
@@ -0,0 +1,9 @@
+import matplotlib
+matplotlib.use("AGG")
+from obspy.core import run_tests
+errors = run_tests(report=True, hostname="conda-forge", verbosity=2)
```

Comment:
> In general I'm a bit paranoid 

I understand that and I am the same. If you can live with the way the tests are run now it is fine. Somehow I had the impression that you would need some extra workarounds.

PS: Travis-CI failure was due to a bad connection. I just re-started and it for you.